v2/authorization_claims.go

```diff
@@ -70,6 +70,7 @@ type ClientTLS struct {
 	Cipher         string       `json:"cipher,omitempty"`
 	Certs          StringList   `json:"certs,omitempty"`
 	VerifiedChains []StringList `json:"verified_chains,omitempty"`
+	ServerName     string       `json:"server_name,omitempty"`
```

Comment:
Should this be HostName? Would that be more descriptive?

Reply:
I used the same terminology as `crypto/tls`: https://pkg.go.dev/crypto/tls#ConnectionState, and I though that in the context of a ClientTLS block, `ServerName` would have a clear meaning.

But I understand that it may be misleading due to the fact that a server also has a `server_name`, so I would like for you to make the choice 😅

Reply:
Good point, we can stick with ServerName or SNI.

Reply:
I think that `SNI` may be the best candidate here. Googling "tls sni" gives pretty relevant results, and it does not reuse an existing field name from other claims data.

If that's ok with you, I can push a commit with this modification.

Reply:
The change would be:

```go
SNI    string  `json:"sni,omitempty"`
```

Reply:
Looping in @philpennock and @ripienaar for their opinions here. I could do either.